tests/nlp/unit/imbu_models_tests.py

```diff
@@ -40,6 +47,94 @@ def setUp(self):
     self.dataPath = os.path.join(TEST_DATA_DIR, "sample_reviews_subset.csv")
 
 
+  def testCreateModel(self):
+
+    # Setup fake ImbuModels instance
+    imbu = ImbuModels(
+      cacheRoot="fake_cache_root",
+      dataPath=self.dataPath,
+      retina="en_associative",
+      apiKey=os.environ.get("CORTICAL_API_KEY")
+    )
+
+    # You must specify a known model type
+    self.assertRaises(Exception, imbu.createModel, "random model type")
+
+    # Assert that you can create models of known types from scratch (i.e. with
+    # empty loadPath value)
+    for modelType in imbu.modelMappings:
+      # You _must_ specify loadPath and savePath.  Assert that attempts to not
+      # specify will result in failure
+      self.assertRaises(Exception, imbu.createModel, modelType)
+      self.assertRaises(Exception, imbu.createModel, modelType, None)
```

Comment:
Not testing for the right exception. Results in *** TypeError: createModel() takes at least 4 arguments (3 given)